app/src/main/java/org/fossasia/susi/ai/rest/responses/others/RelatedTopics.kt

```diff
@@ -0,0 +1,29 @@
+package org.fossasia.susi.ai.rest.responses.others
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+/**
+ * <h1>POJO class to parse related topics object in retrofit response from websearch client.</h1>
+
+ * Created by mayank on 09-12-2016.
+ */
+
+class RelatedTopics
+
+(@SerializedName("FirstURL")
+ @Expose
```

Comment:
https://static.javadoc.io/com.google.code.gson/gson/2.6.2/com/google/gson/annotations/Expose.html

Refer to this